<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MailArchiverWidget</class>
 <widget class="QMainWindow" name="MailArchiverWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>608</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {background-color: rgba(191,191,191,245);}
QPushButton:hover {font: bold; background-color: white; border: 2px solid rgba(240, 240, 240, 252); border-radius: 4px;}
QPushButton:checked{font:bold; background-color: qlineargradient(spread:reflect, x1:1, y1:0, x2:1, y2:0.485, stop:0 rgba(180,180,180,245), stop:1 rgba(191,191,191,245)); border-radius: 4px;}
QPushButton:enabled{border-color: rgba(191,191,191,245);}

QTreeView, QListView, QTableView, QLineEdit {
	background-color: qlineargradient(spread:reflect, x1:1, y1:0, x2:1, y2:0.485, stop:0 rgba(240, 240, 240, 252), stop:1 rgba(255, 255, 255, 255));
border-radius: 3px;
border-width: 1px;
border-color: rgba(191,191, 191, 245);
border-style: solid;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="openArchive">
        <attribute name="title">
         <string>&lt;Open Archive&gt;</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="4">
            <widget class="QPushButton" name="body">
             <property name="text">
              <string>Body</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="6">
            <widget class="QPushButton" name="from">
             <property name="text">
              <string>From</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="full">
             <property name="text">
              <string>Full Message</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QPushButton" name="subject">
             <property name="text">
              <string>Subject</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QToolButton" name="lock">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../res/MailArchiverWidget.qrc">
               <normaloff>:/lock</normaloff>:/lock</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="7">
            <widget class="QListView" name="mailListView">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
            </widget>
           </item>
           <item row="1" column="7">
            <widget class="QPushButton" name="to">
             <property name="text">
              <string>To</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Filter messages by:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1" colspan="6">
            <widget class="QLineEdit" name="searchEdit"/>
           </item>
           <item row="0" column="7">
            <widget class="QPushButton" name="searchButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Search!</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="openMail">
        <attribute name="title">
         <string>&lt;Open EMail&gt;</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QTextEdit" name="headerView">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>84</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>120</height>
             </size>
            </property>
            <property name="undoRedoEnabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
           <widget class="QTextEdit" name="bodyView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>400</height>
             </size>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: qlineargradient(spread:reflect, x1:1, y1:0, x2:1, y2:0.485, stop:0 rgba(240, 240, 240, 252), stop:1 rgba(255, 255, 255, 255));</string>
            </property>
            <property name="undoRedoEnabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Open Archives:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="archivesListView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Folders in current archive:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeView" name="foldersTreeView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>935</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>&amp;New</string>
     </property>
     <addaction name="actionNewTag"/>
     <addaction name="actionNewFolder"/>
     <addaction name="actionNewArchive"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionOpenArchive"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>E&amp;dit</string>
    </property>
    <addaction name="actionEditTags"/>
    <addaction name="actionEditFolder"/>
    <addaction name="separator"/>
    <addaction name="actionEditPreferences"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionArchiveEmails"/>
    <addaction name="actionArchiveEntireFolder"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionEditTags">
   <property name="text">
    <string>&amp;Tags</string>
   </property>
  </action>
  <action name="actionEditFolder">
   <property name="text">
    <string>&amp;Folder</string>
   </property>
  </action>
  <action name="actionEditPreferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionNewFolder">
   <property name="text">
    <string>&amp;Folder</string>
   </property>
  </action>
  <action name="actionNewArchive">
   <property name="text">
    <string>&amp;Archive</string>
   </property>
  </action>
  <action name="actionNewTag">
   <property name="text">
    <string>&amp;Tag</string>
   </property>
  </action>
  <action name="actionOpenArchive">
   <property name="text">
    <string>&amp;Open archive</string>
   </property>
  </action>
  <action name="actionArchiveEmails">
   <property name="text">
    <string>&amp;Archive emails</string>
   </property>
  </action>
  <action name="actionArchiveEntireFolder">
   <property name="text">
    <string>Archive &amp;entire folder</string>
   </property>
  </action>
  <action name="actionExportSelected">
   <property name="text">
    <string>Export Selected Message As [...]</string>
   </property>
  </action>
  <action name="actionViewSelected">
   <property name="text">
    <string>View Selected Message</string>
   </property>
   <property name="toolTip">
    <string>Limited embedded viewer. For better experience export it and open with your mail client.</string>
   </property>
  </action>
  <action name="actionMoveToFolder">
   <property name="text">
    <string>Move To Folder [...]</string>
   </property>
  </action>
  <action name="actionRemoveFromArchive">
   <property name="text">
    <string>Remove File from Archive</string>
   </property>
   <property name="toolTip">
    <string>Permanently deletes the message.</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/MailArchiverWidget.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MailArchiverWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>876</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOpenArchive()</slot>
  <slot>onSelectedOpenedArchive(QModelIndex)</slot>
  <slot>onSelectedFolderOnCurrentArchive(QModelIndex)</slot>
 </slots>
</ui>
